projects/storefrontlib/src/lib/product/components/product-list/container/product-list.component.ts

```diff
@@ -81,6 +81,21 @@ export class ProductListComponent implements OnChanges, OnInit {
     };
 
     this.model$ = this.productSearchService.searchResults$;
+    if (this.model$) {
```

Comment:
I refactored your codes to this:

```
this.model$ = this.productSearchService.searchResults$.pipe(
      tap(searchResult => {
        if (searchResult.breadcrumbs && searchResult.breadcrumbs.length > 0) {
          this.pageTitle = searchResult.breadcrumbs[0].facetValueName;
        } else if (!this.query.includes(':relevance:')) {
          this.pageTitle = this.query;
        }

        if (this.pageTitle) {
          this.pageTitle =
            searchResult.pagination.totalResults +
            ' results for ' +
            this.pageTitle;
        }
      })
    );
```

Reply:
I included a check for null here as there are unit tests that sometimes leave `searchResults$` as null. 

ex. 
```
HeadlessChrome 70.0.3538 (Linux 0.0.0) ProductListComponent in product-list should call ngOnChanges and get search results with params provided with url FAILED
	TypeError: Cannot read property 'pipe' of undefined
```